nym-data-observatory/src/main.rs

```diff
@@ -8,29 +9,44 @@ mod db;
 mod http;
 mod logging;
 
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+struct Args {
+    /// Port to listen on
+    #[arg(long, default_value_t = 8000, env = "NYM_DATA_OBSERVATORY_HTTP_PORT")]
+    http_port: u16,
+
+    /// Path to the environment file
+    #[arg(short, long, default_value = None, env = "NYM_DATA_OBSERVATORY_ENV_FILE")]
+    env_file: Option<String>,
+
+    /// DB connection url
+    #[arg(short, long, default_value = None, env = "NYM_DATA_OBSERVATORY_CONNECTION_URL")]
+    connection_url: Option<String>,
+}
+
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     logging::setup_tracing_logger();
 
-    // if dotenv file is present, load its values
-    // otherwise, default to mainnet
-    setup_env(Some(".env"));
+    let args = Args::parse();
+
+    setup_env(args.env_file); // Defaults to mainnet if empty
```

Comment:
is this comment up to date?

if so, can you write that default in `Args` definition under `default_value`?
```rust
    /// Path to the environment file
    #[arg(short, long, default_value = None, env = "NYM_DATA_OBSERVATORY_ENV_FILE")]
    env_file: Option<String>,
```

Reply:
that's a comment that other files have because `setup_env` defaults to mainnet if `args.env_file` doesn't exist. I'm not sure if it makes sense to write it in the arg description, what do you think?

Reply:
didn't know that, just leave as is then